examples/README.rst

```diff
@@ -85,83 +87,60 @@ Step-by-Step Weather Data Example
 .. code:: scala
 
     import org.apache.spark.h2o._
-    val h2oContext = H2OContext.getOrCreate(spark)
-    import h2oContext._
-    import h2oContext.implicits._
+    val hc = H2OContext.getOrCreate(spark)
+    import hc.implicits._
+    import spark.implicits._
 
-4.  Load weather data for Chicago international airport (ORD), with help
-    from the RDD API:
+4.  Load weather data for Chicago international airport (ORD):
 
 .. code:: scala
 
-    import org.apache.spark.examples.h2o._
     val weatherDataFile = "examples/smalldata/chicago/Chicago_Ohare_International_Airport.csv"
-    val wrawdata = spark.sparkContext.textFile(weatherDataFile,3).cache()
-    val weatherTable = wrawdata.map(_.split(",")).map(row => WeatherParse(row)).filter(!_.isWrongRow())
+    val weatherTable = spark.read.option("header", "true").option("inferSchema", "true").csv(weatherDataFile).withColumn("Date", to_date('Date, "MM/dd/yyyy")).withColumn("Year", year('Date)).withColumn("Month", month('Date)).withColumn("DayofMonth", dayofmonth('Date))
 
 5.  Load airlines data using the H2O parser:
 
 .. code:: scala
 
     import java.io.File
     val dataFile = "examples/smalldata/airlines/allyears2k_headers.zip"
-    val airlinesData = new H2OFrame(new File(dataFile))
+    val airlinesH2OFrame = new H2OFrame(new File(dataFile))
 
 6.  Select flights destined for Chicago (ORD):
 
 .. code:: scala
 
-    val airlinesTable : RDD[Airlines] = asRDD[Airlines](airlinesData)
-    val flightsToORD = airlinesTable.filter(f => f.Dest==Some("ORD"))
+    val airlinesTable = hc.asDataFrame(airlinesH2OFrame)
+    val flightsToORD = airlinesTable.filter('Dest === "ORD")
 
 7.  Compute the number of these flights:
 
 .. code:: scala
 
     flightsToORD.count
 
-8.  Use Spark SQL to join the flight data with the weather data:
+8.  Join the flight and weather data frames:
```

Comment:
`flights`?
just `weather data frame` or `weather data`?

Reply:
what do you mean pls?

Reply:
Here we a dataset with flights and join it with weather data, right?

Reply:
reworded